### PR TITLE
Cherry-pick #9125 to 6.5: Fix division by zero on diskio metricset

### LINKED ISSUE #9124: Unsupported float value (NaN) on diskio events
diskio events are lost as they cannot be serialized because of containing NaN values.
```
2018-11-16T12:34:28.986+0100	WARN	fileout/file.go:127	Failed to serialize the event: unsupported float value: NaN
```

For confirmed bugs, please report:
- Version: At least 6.4.2, 6.4.3, 6.5.0
- Operating System: At least Linux
- Discuss Forum URL: https://discuss.elastic.co/t/system-diskio-obtaining-nan-failed-to-serialize-the-event-unsupported-float-value-nan/156877/3
- Steps to Reproduce: Enable `diskio` metricset

### PR DESCRIPTION
**Pull request by jsoriano**
Cherry-pick of PR #9125 to 6.5 branch. Original message: 

Fixes #9124

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -23,6 +23,8 @@ https://github.com/elastic/beats/compare/v6.5.0...6.5[Check the HEAD diff]
 
 *Metricbeat*
 
+- Fix issue preventing diskio metrics collection for idle disks. {issue}9124[9124] {pull}9125[9125]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: This sounds more like a bug fix then a breaking change?

Comment (jsoriano) [reply CHANGELOG.asciidoc]: And cherry-picked the fix in the changelog here to avoid one more PR.

**Comment by ruflin** [comment]
jenkins, test this

**Comment by jsoriano** [comment]
@ruflin are you ok with merging this?

**Pull request merged**